dr-scripts_install.lic

```diff
@@ -0,0 +1,91 @@
+=begin
+
+	dr-scripts install helper
+	Aims to automate running through the steps from https://github.com/rpherbig/dr-scripts/wiki/First-Time-Setup
+	Please see us on Discord if there are any issues, or raise an issue on Github
+
+        author: Mahtra (mahtra@seul.org)
+        game: Dragonrealms
+        version: 1.0
+
+        changelog:
+                1.0 (2022-01-27):
+                        First release
+
+=end
+
+  def wait_for_script_to_complete(name, args = [], flags = {})
+    script_handle = start_script(name, args.map { |arg| arg =~ /\s/ ?  "\"#{arg}\"" : arg }, flags)
+    if script_handle
+      pause 2
+      pause 0.5 while Script.running.include?(script_handle)
+    end
+    script_handle
+  end
+
+  def download_dependency
+    if(File.exist?('./scripts/dependency.lic')) 
+      echo "Depdency exist already, skipping downloads"
+    else
+      echo "Downloading dependency"
+      wait_for_script_to_complete('repository', ['download', 'dependency.lic'])
```

Comment:
I thought `dependency.lic` was no longer being hosted in `repository`?

Reply:
Actually, it is again.

The reason we wanted it out is that it was tied to one character, which it only technically is now. We figured out there is a password system to it, so @KatoakDR @asechrest or myself can update it (if you give me an email address I can email you so you have it too, or msg me on discord Destahd#4748) that way we're not stuck with the version that's there if one person leaves.

Beyond that, after discussion with folks who deal with newbies on discord, this is the simplest way to setup reliable first-time installs.